### DISCUSSION TITLE
I would like to be able to set reminders/alarms

### DISCUSSION THREAD

**Discussion by scottbourne** [standardnotes/forum#2641] [Suggestions]
...for notes by entering a code, or symbol in front of the date and time I want the alert to go off

**Comment by JaspalSuri** [comment]
Hi @scottbourne, thanks for the feedback! For what it's worth, this feature isn't currently part of our short-term or long-term plans as we're largely focused on improving the note-taking aspects of the apps.

**Reply by thomaseizinger** [reply]
Has this changed in the last two years? I am using standardnotes for many "life admin" things. But the one thing that I can't use it properly for is TODOs with reminders which is a shame.
